entity-framework/core/providers/sql-server/functions.md

```diff
@@ -172,6 +172,8 @@ Math.Floor(d)              | FLOOR(@d)
 Math.Log(d)                | LOG(@d)
 Math.Log(a, newBase)       | LOG(@a, @newBase)
 Math.Log10(d)              | LOG10(@d)
+Math.Max(x, y)             | GREATEST(@x, @y)    | EF Core 9.0
```

Comment:
I'm not sure any more, did we have an inferior translation here before or was it not working at all?

Reply:
not supported before: 
```
The LINQ expression 'DbSet<NullSemanticsEntity>()
    .Where(n => Math.Max(
        val1: n.Int, 
        val2: 14) > 20)' could not be translated. Additional information: Translation of method 'System.Math.Max' failed. 
```